nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/pom.xml

```diff
@@ -169,6 +169,12 @@
             <version>1.18.0-SNAPSHOT</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-standard-record-utils</artifactId>
+            <version>1.18.0-SNAPSHOT</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
Is there a particular reason for the addition of this dependency?

Reply:
I don't think so – I have removed it now; must be an artifact of the coding process.